docs/application/native/guides/app-management/tidl.md

```diff
@@ -29,6 +29,7 @@ Usage:
 > [!NOTE]
 > Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.
 > Generating cion option with `-c` or `--cion` is supported since Tizen 6.5.
+> Generating Rust language type is only supported in protocol version 2.
```

Comment:
It should be specified that it is supported since Tizen 9.0.

Reply:
done

Reply:
done